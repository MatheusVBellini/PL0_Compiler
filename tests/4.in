VAR i;
BEGIN
  WHILE i < 10
    i := i + 1;
  WHILE i < 20 DO
    i := i + 2;
  
END.
