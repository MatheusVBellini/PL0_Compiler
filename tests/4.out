4:5 error: Missing `DO` symbol before this token

Compilation failed, 1 errors
